buildscripts/setup.gradle

```diff
@@ -401,7 +406,24 @@ task keepOrRemoveDependencies {
 
             if (renameConfig.useSqldelightDependencies != true) {
                 println("Removing sqldelight dependencies")
+                removeTextFromFile(
```

Comment:
The reason this does not work is because `removeTextFromFile` does it one line at a time:

```

/**
 * Removes all lines from the given fileName that contain some supplied text.
 */
static def removeTextFromFile(fileName, text) {
    def file = new File(fileName)
    List fileLines = file.readLines()
    file.text = ""
    fileLines.each { line ->
        if (!line.contains(text)) {
            file.append(line)
            file.append("\n")
        }
    }
}
```

We'll need a way to remove an entire block.